src/main/java/javax/validation/Validation.java

```diff
@@ -150,6 +150,8 @@ public static GenericBootstrap byDefaultProvider() {
 	 * {@link ValidationProvider} implementation of the ACME provider.
 	 *
 	 * @param providerType the {@code ValidationProvider} implementation type
+	 * @param <T> the type of the {@code Configuration} corresponding to this {@code ValidationProvider}
+	 * @param <U> the type of the {@code ValidationProvider} implementation
```

Comment:
We do a link only the first time a given type is mentioned in a paragraph. I thought there'd be a note on this at the [contributions guide](http://beanvalidation.org/contribute/coding/) but it seems to be missing. May be worth adding, it's a best practice from the general JavaDoc guidelines published by Sun back in the day.